### ISSUE TITLE
Incorrect type interface for client.task().call() options parameter

### ISSUE THREAD

**Issue by trex-quo** [apify/apify-client-js#261]
I am attempting to run a task programmatically using the given type interface:
```
const run = await apifyClient
   .task(TASK_ID)
   .call(TASK_INPUT, { waitForFinish: 0 });
   
console.log(run.status);
```

and I get the following error:
```
Did not expect property `waitForFinish` to exist, got `0` in object `options`
```

When I instead use the `waitSecs` property mentioned in the JSDoc for this property, the call succeeds and I get the expected output
`RUNNING`

**Comment by mnmkng** [comment]
https://github.com/apify/apify-client-js/blob/69eb8747e65527b245d7ab0bbe6642d1d81cdbc7/src/resource_clients/actor.ts#L108-L126

The type is still wrong here. The options for above, as well as `actor.call()` and maybe in some other places are different from actor start options in the `waitForFinish` vs `waitSecs` parameter.

Pls fix it, it's a tiny thing, but annoying, because it's so confusing. cc @B4nan @vladfrangu I'd do it myself, but I'm going busy with demos.

**Comment by vladfrangu** [comment]
This should've been fixed since 2.6.2, which _seems_ to have recently been released (I pushed a fix for this https://github.com/apify/apify-client-js/commit/1ce8ed5bf0899826c58f248896623f93a18a2a44 a while ago)

**Comment by mnmkng** [comment]
I see. I guess people doing the demos were using old version then. Let's close this. It's a bit of a shame that it took 2 months to release this though 😅
